src/components/home/LogoutButton.tsx

```diff
@@ -1,50 +1,24 @@
 import { FunctionComponent } from "react";
 import { useDispatch, useSelector } from "react-redux";
-import { Button, Stack } from "@mui/material";
 import LogoutIcon from "assets/images/LogoutIcon";
 import { logOut, selectSession } from "modules/session";
-import theme from "theme";
-
-const activeBackground = "rgba(255, 255, 255, 0.1)";
+import SideBarNavLink from "./SideBarNavLink";
 
 const LogoutButton: FunctionComponent = () => {
   const dispatch = useDispatch();
+
   const handleLogout = () => {
     dispatch(logOut());
   };
+
   const { isLoggedIn } = useSelector(selectSession);
 
   return isLoggedIn ? (
-    <Button
+    <SideBarNavLink
```

Comment:
Might just be semantics but does it make sense to change the filename away from Button? Could this also be merged into SideBar.tsx and have it in the same structure as one of the other NavLinks or is it more advisable to have it organised a different way?